src/Raven.Server/Documents/PeriodicBackup/ConcurrentBackupsCounter.cs

```diff
@@ -0,0 +1,79 @@
+using System;
+using Raven.Client.Util;
+using Raven.Server.Commercial;
+
+namespace Raven.Server.Documents.PeriodicBackup
+{
+    public class ConcurrentBackupsCounter
+    {
+        private int _numberOfCoresToUse;
```

Comment:
rename to `_concurrentBackups` or sth like this

Reply:
done